sdks/python/run_pylint.sh

```diff
@@ -39,14 +39,19 @@ EXCLUDED_GENERATED_FILES=(
 
 FILES_TO_IGNORE=""
 for file in "${EXCLUDED_GENERATED_FILES[@]}"; do
-  if [[ $FILES_TO_IGNORE ]]; then
+  if test -n "$FILES_TO_IGNORE"; then
```

Comment:
square brackets in Bash are always subject to confusion, just use `test` thus the reader may type `man test` to check the options